### ISSUE TITLE
Why not finetune backbone lower layer

### ISSUE THREAD

**Issue by jihwanp** [facebookresearch/detr#431]
Thanks for providing the fantastic work.
I've found that in the backbone code, you set gradient computation False for the lower layer(layer1).
Is it intend for better transfer learning? Or if I fine-tune all the backbone layers, does that makes performance worse?
